core/parachain/approval/approval_distribution.cpp

```diff
@@ -1005,15 +1005,6 @@ namespace kagome::parachain {
   void ApprovalDistribution::handle_new_head(const primitives::BlockHash &head,
                                              const network::ExView &updated,
                                              Func &&func) {
-    /*sequenceIgnore(this_context_->template wrap(asAsync(
```

Comment:
Remove or uncomment.
Also clang-format, or move this block to lambda variable (to avoid re-formatting)